### PR TITLE
Xanthus58.VanillaRenewed version 2.2

### PR DESCRIPTION
**Pull request by zephiowl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/49505)

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]
Installs to `C:\Users\WDAGUtilityAccount\Desktop\%appdata%\.minecraft\resourcepacks\Vanilla Renewed`

**Comment by Xanthus58** [comment]
ahh right right forgot about the hash 2 secconds <3

**Comment by Xanthus58** [comment]
Should "Validation-No-Executables" be concerning? I had it working on my end, just confused on how it passed the pipeline but has errors

**Comment by KaranKad** [comment]
`Validation-No-Executables` means it couldn't find any `.exe` after this application is installed, but it makes sense since this is a minecraft resource pack.

**Comment by Xanthus58** [comment]
> `Validation-No-Executables` means it couldn't find any `.exe` after this application is installed, but it makes sense since this is a minecraft resource pack.

Ahhhhh i see thank you <3

File: manifests/x/Xanthus58/VanillaRenewed/2.2/Xanthus58.VanillaRenewed.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Xanthus58.VanillaRenewed
+PackageVersion: 2.2
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/Xanthus58/Vanilla-Renewed/releases/download/1.18.1-2.2/Vanilla.Renewed.2.2.exe
+  InstallerSha256: 053b6361f2e4c5e2b5aaba205d36504425078a82d4a78852537884efa43c777e
```
Comment (ItzLevvie) [inline manifests/x/Xanthus58/VanillaRenewed/2.2/Xanthus58.VanillaRenewed.installer.yaml:10]:
```suggestion
  InstallerSha256: 053b6361f2e4c5e2b5aaba205d36504425078a82d4a78852537884efa43c777e
  Scope: user
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**